NoSQL/README-sec.md

```diff
@@ -0,0 +1,423 @@
+# Oracle NoSQL Database on Docker
+
+Sample Docker build files to facilitate installation and environment setup for
+DevOps users. For more information about Oracle NoSQL Database please see the
+[Oracle NoSQL Database documentation][DOCS].
+
+This project offers sample container image configuration files for:
+
+* [Oracle NoSQL Database Community Edition](ce/Dockerfile)
+
+This container image uses a simplified version of the Oracle NoSQL Database called
+ KVLite. KVLite runs as a single process that provides a single storage node and
+ single storage shard. KVLite does not include replication or administration.
+
+This container image configures an Oracle NoSQL Database secure configuration
```

Comment:
Why would we want to continue hosting a non-secure version? Why not replace the existing image with this secure version instead? Oracle best practice for releasing sample/demo code would require this anyway.

Reply:
@mikebrey comments? My understanding was that we wanted to maintain the two versions.
- Oracle NoSQL Database **secure** configuration
- Oracle NoSQL Database **non-secure** configuration

Reply:
Maintaining a non-secure version would violate more Oracle software assurance and release policies than I care to consider, so if this is your intention, I'd need to get sign off from your security lead and from other folks in release management before merging.

Reply:
@Djelibeybi  The docker container that we are releasing is KVLITE.  KVLITE is a single process version of NoSQL for testing purposes only.     It is specifically designed for a single user.   With that being said, it can be configured in a secure or a non-secure mode similar to the NoSQL product itself.   We got feedback from several customers wanting to use KVLITE as a way to understand how to set up the real database in a secure manner.    Within docker we provide the scripts that get executed to set up KVLITE in a secure manner.  The concepts in those scripts can be used to set up the actual NoSQL database in a secure manner.    For the product itself, it complies with the security guidelines and we release with the defaults for a secure configuration.    KVLITE, as a testing tool, is a way for customers to test their application without having to set up a real database.   KVLITE is not the product.   If based on this, you still feel the need to get security lead sign off, I can pursue that.

Reply:
Oracle policy around security is generally pretty strict in this regard, and given the use of the terms "secure" and "non-secure" to describe each image intended for public release/use and because I'm not qualified to determine the security posture of KVLite in any configuration, I'd prefer to get sign-off from your security lead to make sure that even the non-secure image complies with the required Oracle security policy requirements for release, with the secure option adding additional security on top of the minimum requiement.

I've switched this PR to draft until we connect internally and the security review and sign-off is completed.

Reply:
@Djelibeybi  Avi - I understand the position you are taking.    I will discuss with our security lead in database.